src/couch_replicator/src/couch_replicator_docs.erl

```diff
@@ -695,7 +695,8 @@ strip_credentials(Url) when is_binary(Url) ->
         "http\\1://\\2",
         [{return, binary}]);
 strip_credentials({Props}) ->
-    {lists:keydelete(<<"oauth">>, 1, Props)}.
+    Props0 = lists:keydelete(<<"oauth">>, 1, Props),
```

Comment:
There are a few unit tests below, let's add a few tests for the new code as well.

Reply:
Yes, added `check_strip_credentials_test`